pkg/api/vlabs/types.go

```diff
@@ -38,6 +38,7 @@ type Properties struct {
 	MasterProfile           MasterProfile           `json:"masterProfile"`
 	AgentPoolProfiles       []AgentPoolProfile      `json:"agentPoolProfiles"`
 	LinuxProfile            LinuxProfile            `json:"linuxProfile"`
+	Environment				Environment				`json:"environment"`
```

Comment:
Environment can be derived by 'Location' field in ContainerService struct. I don't think a separate field 'Environment' is needed in properties for this.